dataikuapi/dss/modelevaluationstore.py

```diff
@@ -110,32 +110,42 @@ def delete(self):
 
 
     ########################################################
-    # Runs
+    # Model evaluations
     ########################################################
 
-    def list_model_evaluations(self, as_type=None):
+    def list_model_evaluations(self):
         """
-        List the model evaluations in this model evaluation store.
+        List the model evaluations in this model evaluation store. The list is sorted
+        by ME creation date.
 
         :returns: The list of the model evaluations
-        :rtype: list
+        :rtype: list of :class:`dataikuapi.dss.modelevaluationstore.DSSModelEvaluation`
         """
         items = self.client._perform_json("GET", "/projects/%s/modelevaluationstores/%s/runs/" % (self.project_key, self.mes_id))
-        if as_type in ["objects", "object"]:
-            return [DSSModelEvaluation(self, item["ref"]["runId"]) for item in items]
-        else:
-            return items
+        return [DSSModelEvaluation(self, item["ref"]["runId"], item["created"]) for item in items]
 
     def get_model_evaluation(self, run_id):
         """
-        Get a handle to interact with a specific model evaluations
+        Get a handle to interact with a specific model evaluation
        
         :param string run_id: the id of the desired model evaluation
         
         :returns: A :class:`dataikuapi.dss.modelevaluationstore.DSSModelEvaluation` model evaluation handle
         """
         return DSSModelEvaluation(self, run_id)
 
+    def get_latest_model_evaluation(self):
+        """
+        Get a handle to interact with the latest model evaluation computed
+
+
+        :returns: A :class:`dataikuapi.dss.modelevaluationstore.DSSModelEvaluation` model evaluation handle
+        """
+
+        latest_run_id = self.client._perform_text(
+            "GET", "/projects/%s/modelevaluationstores/%s/latestRunId" % (self.project_key, self.mes_id))
+        return DSSModelEvaluation(self, latest_run_id)
```

Comment:
Should not a `DSSModelEvaluation` if the MES is empty (in this case, the `run_id` seems to be "")

Reply:
ok, returns none when there is no latest_run_id